cli/src/serve.js

```diff
@@ -229,6 +229,11 @@ const initialize_servers = (ctor, opts) => {
           console.info(`Static files being served from ${opts.serve_static}`);
         }
         srv.on('request', fileServer(opts.serve_static));
+      } else {
```

Comment:
Although it looks like the server should be 404ing the requests already, so that's a little strange.